src/arrayMethodSort.js

```diff
@@ -4,8 +4,39 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function(
+    compareFunction = (firstItem, secondItem) => {
+      const firstItemStr = '' + firstItem;
+      const secondItemStr = '' + secondItem;
+
+      if (firstItemStr > secondItemStr) {
+        return 1;
+      } else if ((firstItemStr < secondItemStr)) {
+        return -1;
+      }
+
+      return 0;
+    }) {
```

Comment:
```suggestion
    compareFunction = compareByDefault) {
```
А compareByDefault объяви вне applyCustomSort